src/quick-setup.md

```diff
@@ -61,15 +61,15 @@ Bear in mind, if you plan on using the server purely for [standalone usage via O
 1. Figure out which body parts you will need to assign. Depending on how many trackers you have these are the suggested locations:
 
 	* Lower-Body Set (5 trackers) - Chest, both thighs, both ankles.
-	* Core Set (5 trackers with one extension) - Chest and hip (via the tracker with an extension), both thighs, both ankles.
-	* Enhanced Core Set (5 trackers with three extensions) - Chest and hip (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).
-	* Full-Body Set (7 trackers with three extensions) - Both upper arms, chest and hip (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).
+	* Core Set (5 trackers with one extension) - Chest and hip/waist (via the tracker with an extension), both thighs, both ankles.
+	* Enhanced Core Set (5 trackers with three extensions) - Chest and hip/waist (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).
+	* Full-Body Set (7 trackers with three extensions) - Both upper arms, chest and hip/waist (via trackers with extension), both thighs, both ankles and feet (via trackers with extensions).
```

Comment:
This isn't necessarily wrong, but should we recommend waist? I've found hip performs better in general, but this is kinda a personal opinion thing when it comes to this.

Reply:
I do agree that hip performs better, but there were people in the Discord who struggled to get good mounting on hip and weren't aware that you can use waist instead

Reply:
Understandable, I just think people might be attracted towards using "waist" instead as they know "waist tracking" from SteamVR stuff. Either way this is fine, I'll just leave this up for consideration but this isn't really a big issue.